cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/workflow/WorkflowDriver.java

```diff
@@ -375,11 +375,11 @@ private void executeFork(final ApplicationSpecification appSpec, WorkflowForkNod
           String branchInfo = retValue.getKey();
           WorkflowToken branchToken = retValue.getValue();
           ((BasicWorkflowToken) token).mergeToken(branchToken);
-          LOG.info("Execution of branch {} for fork {} completed", branchInfo, fork);
+          LOG.trace("Execution of branch {} for fork {} completed", branchInfo, fork);
         } catch (Throwable t) {
           Throwable rootCause = Throwables.getRootCause(t);
           if (rootCause instanceof ExecutionException) {
-            LOG.error("Exception occurred in the execution of the fork node {}.", fork, rootCause);
+            LOG.error("Exception occurred in the execution of the fork node {}.", fork);
             throw (ExecutionException) rootCause;
           }
           if (rootCause instanceof InterruptedException) {
```

Comment:
shouldn't log error. You can log trace for the interrupted exception due to execution abortion.